.all-contributorsrc

```diff
@@ -71,6 +71,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
```

Comment:
There is no need to add yourself to the Contributions list, this is an automated job which after merging the PR, I'll do it just by a comment. So, **rollback the changes on this file**.